README.md

```diff
@@ -0,0 +1,19 @@
+# Intel Retail Documentation Repository
```

Comment:
Will this repo only contain documentation ?

Reply:
Yes, only design documentation and other relevant docs.